app/clients/sms/aws_pinpoint.py

```diff
@@ -97,6 +98,10 @@ def _validate_response(
             error_message = f'StatusCode: {result["StatusCode"]}, StatusMessage:{result["StatusMessage"]}'
 
             if delivery_status in ['DUPLICATE', 'OPT_OUT', 'PERMANENT_FAILURE']:
+                # indicates 'From' number doesn't exist for this Pinpoint account
+                if 'provided number does not exis' in result['StatusMessage']:
```

Comment:
Type: change `exis` to `exist`

Reply:
it should still work the same, but good catch!